### PR TITLE
More docs

### PR DESCRIPTION
**Pull request by emulbreh**

### CODE REVIEW

File: docs/api/config_api.rst [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    .. attribute:: root
+    
+        A reference to the root :class:`Configuration` instance.
+
+
+.. class:: Configuration(values=None)
+
+    :param values: an optional initial mapping
+
+    Configuration implements the same onterface as :class:`ConfigView` in addition
```
Comment (jacqueslh) [inline docs/api/config_api.rst]: `the same onterface`

Comment (jacqueslh) [inline docs/api/config_api.rst]: just wondering, should this not be docstring in the code ?

**Comment by mouadino** [comment]
LGTM

File: docs/api/config_api.rst (hunk 1 above)
Comment (emulbreh) [inline docs/api/config_api.rst]: no, docstrings aren't documentation (for me).

**Pull request merged**